### PR TITLE
Improved Embeds.md

### PR DESCRIPTION
**Pull request by YT-GameWorks**
The original one was too vague, and unclear so I put in 2 sections describing some of the methods

### CODE REVIEW

File: docs/wiki/basic-tutorials/embeds.md [hunk 1]
```diff
@@ -39,6 +39,34 @@ EmbedBuilder embed = new EmbedBuilder()
 // Send the embed
 channel.sendMessage(embed);
 ```
+All this is a bit confusing :thinking: so let's break it down!
+
+## :bone: Basics
+To start using the embed builder, you need to instantiate it like any other class and then you can start acessing it's methods. We'll go through the basic ones here:
```
Comment (Joshix-1) [inline docs/wiki/basic-tutorials/embeds.md:45]: *accessing

File: docs/wiki/basic-tutorials/embeds.md [hunk 2]
```diff
@@ -39,6 +39,34 @@ EmbedBuilder embed = new EmbedBuilder()
...
+
+* `EmbedBuilder.setTitle("")`- Most of the time you will want a title for your embed, won't you :question: This sets the title of the embed, the top-most words in the embed.
```
Comment (Joshix-1) [inline docs/wiki/basic-tutorials/embeds.md:47]: EmbedBuilder#setTitle() is like the other methods not static. You shouldn't use a dot there.

File: docs/wiki/basic-tutorials/embeds.md [hunk 3]
```diff
@@ -39,6 +39,34 @@ EmbedBuilder embed = new EmbedBuilder()
...
+
+* `EmbedBuilder.setDescription("")`- This sets the description, this is like the subtitle of your embed if you will. It is displayed right after the title.
+
+* `EmbedBuilder.addFeild("", "", false)`- So if you want more subtitles than your description, This is the way to do that! thats cool and all, but what are the 3 parameters? The first one is the subtitle of the feild and the second one is the content of that subtitle, and the third one is a boolean that will change the position of the subtitle, suppose there are 2 of these "feilds" and you want them to go side by side then just set both (or any one) to have the inline value (the last parameter) to true and it will display it on the same line! Alternatively you can also use the dedicated method, `EmbedBuilder.addInlineFeild("");` to do the work of the last parameter in the `EmbedBuilder.addFeild("")`
```
Comment (Joshix-1) [inline docs/wiki/basic-tutorials/embeds.md:51]: it's field not feild

File: docs/wiki/basic-tutorials/embeds.md (hunk 2 above)
Comment (Joshix-1) [reply docs/wiki/basic-tutorials/embeds.md:47]: And I don't think you should use "" as a placeholder for Strings. You can't even set it to an empty String.

File: docs/wiki/basic-tutorials/embeds.md (hunk 3 above)
Comment (Joshix-1) [inline docs/wiki/basic-tutorials/embeds.md:51]: Maybe mention here that one can achieve empty fields with invisible characters like "\u200B"

File: docs/wiki/basic-tutorials/embeds.md [hunk 4]
```diff
@@ -39,6 +39,34 @@ EmbedBuilder embed = new EmbedBuilder()
...
+
+* `EmbedBuilder.setFooter("", "")`- This sets the bottom-most text in the embed, There is also an optional last parameter, which is a url for a image that will be displayed beside the text at the bottom
+
+* `EmbedBuilder.setColor(Color)`- This setts the color on the side of the embed... The way you use it is create a new color class, like so: `Color embedColor = new Color(r, g, b)` and put the RGB (ints) values in the constructor, and then use that variable in the `setColor()`. One neat perk, if you use intellij idea is that you have a integrated color picker when you use the constructor to set the color. Alternatively you can also use the predefined colors like a `Color.RED` and many others.
```
Comment (Joshix-1) [inline docs/wiki/basic-tutorials/embeds.md:55]: *sets with one t

Comment (Joshix-1) [inline docs/wiki/basic-tutorials/embeds.md:55]: I don't think we should explain basic java classes in this.

Comment (Joshix-1) [inline docs/wiki/basic-tutorials/embeds.md:55]: It is IntelliJ IDEA, but I don't think we should mention this here.

File: docs/wiki/basic-tutorials/embeds.md [hunk 5]
```diff
@@ -39,6 +39,34 @@ EmbedBuilder embed = new EmbedBuilder()
...
+
+* `EmbedBuilder.setFooter("", "")`- This sets the bottom-most text in the embed, There is also an optional last parameter, which is a url for a image that will be displayed beside the text at the bottom
+
+* `EmbedBuilder.setColor(Color)`- This setts the color on the side of the embed... The way you use it is create a new color class, like so: `Color embedColor = new Color(r, g, b)` and put the RGB (ints) values in the constructor, and then use that variable in the `setColor()`. One neat perk, if you use intellij idea is that you have a integrated color picker when you use the constructor to set the color. Alternatively you can also use the predefined colors like a `Color.RED` and many others. 
+
+## :gear: Advanced topics
+These are some of the more rarely used but still useful methods:
+
+* `EmbedBuilder.setImage("")`- Adds an Image to the bottom of the embed, This can edither be a link to a image on the web or one stored on your computer.
```
Comment (Joshix-1) [inline docs/wiki/basic-tutorials/embeds.md:60]: edither → either

File: docs/wiki/basic-tutorials/embeds.md [hunk 6]
```diff
@@ -39,6 +39,34 @@ EmbedBuilder embed = new EmbedBuilder()
...
+
+* `EmbedBuilder.setThumbnail("")`- Adds an image to the embed but this time at the right side of it! aThis can either be a link or a local file.
```
Comment (Joshix-1) [inline docs/wiki/basic-tutorials/embeds.md:62]: aThis → This

File: docs/wiki/basic-tutorials/embeds.md [hunk 7]
```diff
@@ -39,6 +39,34 @@ EmbedBuilder embed = new EmbedBuilder()
...
+
+* `EmbedBuilder.setAuthor("", "")`- This adds text to the top of the embed (above the title), you can make it dynamic by using `event.getMessageAuthor().getDisplayName();`, It has an optional second parameter which can be an image much like the footer.
```
Comment (Joshix-1) [inline docs/wiki/basic-tutorials/embeds.md:64]: Shouldn't it be clear that you can put any Strings here?

File: docs/wiki/basic-tutorials/embeds.md [hunk 8]
```diff
@@ -39,6 +39,34 @@ EmbedBuilder embed = new EmbedBuilder()
...
+
+* `EmbedBuilder.setTimestampToNow()`- Sets the Timestamp of the embed to be the current time.
+
+* `EmbedBuilder.getTimestamp()`- gets the timesStamp of the embed, useful for when you have to add a footer and set it to the time the message was sent.
```
Comment (Joshix-1) [inline docs/wiki/basic-tutorials/embeds.md:68]: This is no method.

File: docs/wiki/basic-tutorials/embeds.md (hunk 5 above)
Comment (Joshix-1) [reply docs/wiki/basic-tutorials/embeds.md:60]: The String in EmbedBuilder#setImage(String url) can only be an url to an external image. To send local images you need one of the other setImage methods

**Review by Joshix-1 — CHANGES_REQUESTED** [review]
I don't think we need this at all, because the current explains it enough in combination with the docs.
But here are some suggestions on things you can improve.

**Comment by YT-GameWorks** [comment]
Hello, thanks for pointing out the mistakes, it is my first time writing a wiki, also I think we really need this as a beginner I was confused by the wiki with just a embed example with no detailed explanation, at least explain the more complex ones! I know a lot of them are self explanatory but I was confused by it a lot... Anyway I will continue to improve this...

**Comment by Bastian** [comment]
Hi, thank you for the Pull Request!

I have to agree with Joshix, that it is probably not a good idea to just explain every single method, as we also don't do it in any of the other wiki articles.

What do you think about adding information to the example image itself like this:
image

**Pull request closed without merging**